pages/common/iperf.md

```diff
@@ -0,0 +1,15 @@
+# iperf
+
+> Measure network bandwidth between computers.
+
+- Run on server:
+
+`iperf -s`
+
+- Run on client:
+
+`iperf -c {{server_address}}`
```

Comment:
What is meant by `server_address`? Is it just an IP address, or a port number too?

Reply:
Just an IP address.  If no port number is provided, iperf uses the default port 5001.

Reply:
Ah, ok. I think this should be fine then :-)